test/index.js

```diff
@@ -1,245 +0,0 @@
-require('should');
-var Limiter = require('..');
```

Comment:
You probably noticed that the old `get` method is now read only, providing a way to get the rate-limiting status for the targeted id without increasing the hits.

Use case: https://dev.twitter.com/rest/reference/get/application/rate_limit_status.

Is it ok for you ?